contracts.proto

```diff
@@ -10,6 +10,7 @@ enum KnownType {
   Guid = 3;
   Uri = 4;
   Boolean = 5;
+  Binary = 6;
```

Comment:
Or should I start counting from 50 to save some space for more `System` types in the future here?

Reply:
I would append below the Operation.